.circleci/config.yml

```diff
@@ -101,21 +101,21 @@ commands:
       - run: make fmt-check
 
 jobs:
-  test-llvm11-go115:
+  test-llvm11-go116:
```

Comment:
IMHO we should either drop support for Go 1.15 entirely or continue to support in including in CI. We can for example test Go 1.15 and 1.17: it's unlikely that with those tests, Go 1.16 will somehow break.
(I don't have a strong preference, but unless it holds back some new features I'd be in favor of keeping Go 1.15 support for now).

Reply:
I changed the 1.16 to 1.15 in circleci.